src/controllers/exams.controller.js

```diff
@@ -69,9 +69,25 @@ const modifyExam = asyncHandler(async (req, res, next) => {
       });
 });
 
+const addExam = asyncHandler(async (req,res,next) => {
+    const { name, description } = req.body;
 
+    const exam = new Exam({ 
+        name : name,
+        description : description,
+    })          
+    
+    await exam.save();
+
+    // const question = await Question.create({ email, password, firstname, lastname })
+
+    res.status(200).json({
```

Comment:
for status codes 
you can import 
`const { StatusCodes } = require('http-status-codes');
res.status(StatusCodes.OK).json({ success: true, message: 'Exam added successfully', exam })
`